package.json

```diff
@@ -40,7 +40,6 @@
     "grunt-contrib-uglify": "^1.0.1",
     "grunt-contrib-watch": "^1.0.0",
     "grunt-conventional-changelog": "^6.1.0",
-    "grunt-ddescribe-iit": "0.0.6",
```

Comment:
This plugin prevents you from focusing a single test at a time to prevent contributors from accidentally committing code with only a subset of tests being run. This updates removes this restriction so we don't have to run the ENTIRE test suite at once (which takes a while because it does DOM testing WITHOUT a headless browser).